.travis.yml

```diff
@@ -50,6 +53,10 @@ before_install:
     $GCLOUD/gcloud auth activate-service-account --key servicekey.json;
   fi
 - R --version
+# use new gcc version
+- sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50
```

Comment:
Adding sudo moves us out of the travis container infrastructure and back to their non-dockerized infrastructure.  This used to be a bad thing since they had a very limited number of machines.  It looks like they've now improved their non-dockerized infrastructure to be properly elastic.  So this should be fine.